api/pkg/webhooks/manager.go

```diff
@@ -0,0 +1,208 @@
+package webhooks
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+type WebhookManager interface {
+	InvokeWebhooks(
+		context.Context,
```

Comment:
can we add var name so they don't appear as `func([]byte) error` which is quite hard to infer from the caller

Reply:
Fixed